TombEngine/Scripting/Internal/ReservedScriptNames.h

```diff
@@ -238,13 +238,15 @@ static constexpr char ScriptReserved_GetWeaponType[]				= "GetWeaponType";
 static constexpr char ScriptReserved_ThrowAwayTorch[]				= "ThrowAwayTorch";
 static constexpr char ScriptReserved_SetWeaponType[]				= "SetWeaponType";
 static constexpr char ScriptReserved_TorchIsLit[]					= "TorchIsLit";
+static constexpr char ScriptReserved_Log[]							= "Log";
```

Comment:
I would suggest `PrintLog` or something similar because the function is rather ambiguous otherwise.